autorest/app.ts

```diff
@@ -81,6 +84,9 @@ function parseArgs(autorestArgs: Array<string>): any {
         value = rawValue;
       }
       result[key] = value;
+    } else {
+      result._ = result._ || [];
```

Comment:
Where does this get used?  Doesn't seem to be used by the other changes in the PR